documentation/asciidoc/computers/os/updating.adoc

```diff
@@ -42,9 +42,9 @@ When running `sudo apt full-upgrade`, it will show how much data will be downloa
 
 ==== Upgrading from Previous Operating System Versions
 
-WARNING: Upgrading an existing image is possible, but is not guaranteed to work in every circumstance and we do not recommend it. If you do wish to try upgrading your operating system version, we strongly suggest making a backup first -- we can accept no responsibility for loss of data from a failed update.
+WARNING: Upgrading an existing image is sometimes possible, but is not guaranteed to work in every circumstance and we do not recommend it. If you do wish to try upgrading your operating system version, we strongly suggest making a backup first. We accept no responsibility for the loss of any data during a failed update.
 
-The latest version of Raspberry Pi OS is based on https://www.raspberrypi.com/news/raspberry-pi-os-debian-bullseye/[Debian Bullseye]. The previous version was based on https://www.raspberrypi.com/news/buster-the-new-version-of-raspbian/[Buster]. If you want to perform an in-place upgrade from Buster to Bullseye (and you're aware of the risks) see the https://forums.raspberrypi.com/viewtopic.php?t=323279[instructions in the forums].
+The latest version of Raspberry Pi OS is based on https://link.tbd[Debian Bookworm]. The previous version was based on https://www.raspberrypi.com/news/raspberry-pi-os-debian-bullseye/[Debian Bullseye].
```

Comment:
If we just don't link `https://link.tbd[Debian Bookworm]` here we can always go in later and do it post launch. If we do and forget then we have a broken link. So just yank the link here.